src/pages/Graph/SummaryPanelGraph.tsx

```diff
@@ -331,29 +337,29 @@ export default class SummaryPanelGraph extends React.Component<SummaryPanelPropT
       <br />
       {numApps > 0 && (
         <>
-          <Icon name="applications" type="pf" style={{ padding: '0 1em' }} />
+          <KialiIcon.Applications className={topologyStyle} />
           {numApps.toString()} {numApps === 1 ? 'app ' : 'apps '}
           {numVersions > 0 && `(${numVersions} versions)`}
           <br />
         </>
       )}
       {numSvc > 0 && (
         <>
-          <Icon name="service" type="pf" style={{ padding: '0 1em' }} />
+          <KialiIcon.Services className={topologyStyle} />
           {numSvc.toString()} {numSvc === 1 ? 'service' : 'services'}
           <br />
         </>
       )}
       {numWorkloads > 0 && (
         <>
-          <Icon name="bundle" type="pf" style={{ padding: '0 1em' }} />
+          <KialiIcon.Workloads className={topologyStyle} />
           {numWorkloads.toString()} {numWorkloads === 1 ? 'workload' : 'workloads'}
           <br />
         </>
       )}
       {numEdges > 0 && (
         <>
-          <Icon name="topology" type="pf" style={{ padding: '0 1em' }} />
+          <KialiIcon.Topology />
```

Comment:
The topologyStyle className is missing here. I think this is why it has no padding :)

![image](https://user-images.githubusercontent.com/23639005/67606508-37a7e700-f747-11e9-8d70-d164acf94fbc.png)

Reply:
Yes, good catch!